### PR TITLE
Add National indicators feature, refs: #17023

### PR DESCRIPTION
**Pull request by anaion**
Please review and merge.

### CODE REVIEW

File: src/nbsap/forms.py [hunk 1]
```diff
@@ -450,6 +450,57 @@ def save(self):
         return indicator
 
 
+class NationalIndicatorForm(forms.Form):
+    language = forms.ChoiceField(choices=settings.LANGUAGES)
+    title = forms.CharField(widget=widgets.Textarea, required=False)
```
Comment (alexef) [inline src/nbsap/forms.py]: The title should be required.

File: src/nbsap/forms.py [hunk 2]
```diff
@@ -483,6 +534,39 @@ def save(self):
         return indicator
 
 
+class NationalIndicatorEditForm(NationalIndicatorForm, ChoicesMixin):
+    code = forms.CharField(max_length=16, validators=[validate_eu_target_code],
```
Comment (alexef) [inline src/nbsap/forms.py]: Another validator should be used, or at least change the name.

File: src/nbsap/views/nat_indicators.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    })
+
+
+@auth_required
+def delete_nat_indicator(request, pk):
+    if request.method == 'POST':
+        ind = get_object_or_404(models.NationalIndicator, pk=pk)
+        ind.delete()
+        messages.success(request, _('Indicator successfully deleted.') + "")
+        return redirect('list_nat_indicators')
```
Comment (alexef) [inline src/nbsap/views/nat_indicators.py]: Else

File: src/nbsap/views/nat_indicators.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    return render(request, 'manager/nat_indicators/view_nat_indicator.html', {
+        'indicator': ind,
+    })
+
+
+@auth_required
+def edit_nat_indicator(request, pk=None):
+    if pk:
+        indicator = get_object_or_404(models.NationalIndicator, pk=pk)
+        print indicator
```
Comment (alexef) [inline src/nbsap/views/nat_indicators.py]: remove print

File: src/nbsap/forms.py (hunk 2 above)
Comment (anaion) [inline src/nbsap/forms.py]: How about we remove the code field completely? It was anyway added to the NationalIndicatorForm.

File: src/nbsap/templates/manager/nat_indicators/edit_nat_indicator.html [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+{% endblock %}
+
+
+{% block scripts %}
+  <script src="/static/js/language_change.js"></script>
+  {% assets "js_admin" %}
+    <script type="text/javascript" src="{{ ASSET_URL }}"></script>
+  {% endassets %}
+
+{#  <script>#}
```
Comment (alexef) [inline src/nbsap/templates/manager/nat_indicators/edit_nat_indicator.html]: Does this commented block need to stay?

Comment (anaion) [inline src/nbsap/templates/manager/nat_indicators/edit_nat_indicator.html]: No, it can definitely go away. Thanks!

**Comment by alexef** [comment]
Lgtm, ok to merge.

**Comment by alexef** [comment]
@anaion rebased against master, tests are failing, can you please review, fix and merge?

**Pull request merged**